### PR TITLE
IGAPP-305: Prepare and translate override translations

### PR DESCRIPTION
**Pull request by maxammann**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

This is only configuration and preparing. The translations will follow in a separate PR.

### CODE REVIEW

**Comment by maxammann** [comment]
The translation files are here: https://drive.google.com/drive/folders/1gimQDlovKYpFASRX5_GMfbF5Nqc-MCBM?usp=sharing

**Review by klinzo — APPROVED** [review]

File: translations/tools/manage.js [hunk 1]
```diff
@@ -202,17 +212,28 @@ program
...
-      }
-    } else if (targetFormat === 'json') {
-      writeJsonFromCsv(fromPath, toPath, sourceLanguage)
+      }],
+      ['csv-json', () => {
+        writeJsonFromCsv(fromPath, toPath, sourceLanguage)
+      }]
+    ])
+
+    const convert = converter.get(`${sourceFormat.toLowerCase()}-${targetFormat.toLowerCase()}`)
```
Comment (Znippet) [inline translations/tools/manage.js:230]: Sweet, didn't know they have Map in ES6 until now

**Review by Znippet — APPROVED** [review]

File: translations/tools/manage.js (hunk 1 above)
Comment (maxammann) [reply translations/tools/manage.js:230]: Also use them with caution. This is a bad use it it actually :D first wanted to make the key an array. But it does not work.

Will revert this to a plain object.

**Comment by SteffiStuffel** [comment]
I tried it, but i guess i'm missing something.

I exported translations with 
yarn export:ods:translations

edited the en.ods in the third line (changed Events to EventsEn)

imported it with  
yarn import:ods:translations

exits properly without errors. translation.json is updated, but neither native (although reinstalled) nor webapp is showing 'EventsEN' in the heading?

**Pull request merged**